src/app/screens/Discover/websites.json

```diff
@@ -153,6 +153,12 @@
         "subtitle": "The blue bird experience for Nostr",
         "logo": "https://cdn.getalby-assets.com/alby-extension-website-screen/yosup_thumbnail.svg",
         "url": "https://yosup.app/"
+      },
+      {
+        "title": "Primal",
+        "subtitle": "Nostr highlights by Primal",
```

Comment:
I don't think this subtitle is useful. If there's no good subtitle can it be just empty?